yt/yt/server/queue_agent/config.h

```diff
@@ -159,6 +180,9 @@ class TQueueAgentDynamicConfig
     //! Configuration of queue controllers.
     TQueueControllerDynamicConfigPtr Controller;
 
+    //! Configuration of queue static table export manager.
+    TQueueStaticTableExportManagerDynamicConfig QueueStaticTableExportManager;
```

Comment:
Let's store it as intrusive ptr?
`TQueueStaticTableExportManagerDynamicConfigPtr QueueStaticTableExportManager`